mesheryctl/internal/cli/root/pattern/error.go

```diff
@@ -39,3 +60,60 @@ func ErrPatternInvalidNameOrID(err error) error {
 		[]string{"Invalid pattern name or ID"},
 		[]string{"Run `mesheryctl pattern view -a` to view all available patterns."})
 }
+
+func ErrImportpattern(err error) error {
```

Comment:
Follow better naming conventions of error functions like below...
```suggestion
func ErrImportPattern(err error) error {
```

This applies to every function and error code necessary. So, I'm not marking every single one of them in this review. Make changes wherever this applies

Reply:
Typo Fix. ✅